client/packages/common/src/ui/layout/tables/material-react-table/useMaterialTableColumns.tsx

```diff
@@ -7,95 +7,100 @@
  * current URL query into the filter state required by MRT.
  */
 import { useMemo } from 'react';
-import { MRT_ColumnDef, MRT_RowData } from 'material-react-table';
+import { MRT_RowData } from 'material-react-table';
 import {
-  ColumnDef,
   DateUtils,
   mergeCellProps,
   useFormatDateTime,
+  useSimplifiedTabletUI,
   useUrlQuery,
 } from '@openmsupply-client/common';
 
+import { ColumnDef } from './types';
+
 export const useMaterialTableColumns = <T extends MRT_RowData>(
-  columns: ColumnDef<T>[]
+  omsColumns: ColumnDef<T>[]
 ) => {
-  const { urlQuery, updateQuery } = useUrlQuery();
-  const { customDate, urlQueryDateTime } = useFormatDateTime();
-
-  const { mrtColumnDefinitions, filterUpdaters } = useMemo(() => {
-    const mrtColumnDefinitions: MRT_ColumnDef<T>[] = [];
-    const filterUpdaters: Record<string, (value: any) => void> = {};
-
-    columns.forEach(
-      ({ filterType = 'none', filterValues, align, ...mrtProperties }) => {
-        const filterKey = (mrtProperties.accessorKey ||
-          mrtProperties.id) as string;
-
-        const columnDefProperties: Partial<MRT_ColumnDef<T>> = {};
-
-        // Add canonical filter definitions
-        switch (filterType) {
-          case 'none':
-            columnDefProperties.enableColumnFilter = false;
-            break;
-          case 'text':
-            filterUpdaters[filterKey] = (value: string) => {
-              updateQuery({ [filterKey]: value });
-            };
-            break;
-
-          case 'enum':
-            columnDefProperties.filterVariant = 'select';
-            columnDefProperties.filterSelectOptions = filterValues;
-            filterUpdaters[filterKey] = (value: string) => {
-              updateQuery({ [filterKey]: value });
-            };
-            break;
-          // case 'number':
-          //   TO-DO
-          case 'dateRange':
-            columnDefProperties.filterVariant = 'date-range';
-            filterUpdaters[filterKey] = ([date1, date2]: [
-              Date | '',
-              Date | '',
-            ]) => {
-              updateQuery({
-                [filterKey]: {
-                  from: date1 ? customDate(date1, urlQueryDateTime) : '',
-                  to: date2 ? customDate(date2, urlQueryDateTime) : '',
-                },
-              });
-            };
-            break;
-        }
-
-        // Add alignment styling
-        if (align) {
-          columnDefProperties.muiTableBodyCellProps = params =>
-            mergeCellProps(
+  const simplifiedMobileView = useSimplifiedTabletUI();
+  const tableDefinition = useMemo(() => {
+    const columns = omsColumns
+      .filter(col => col.includeColumn !== false)
```

Comment:
if `include` is false, don't include in the column definitions passed into the table at all. Essentially does the same thing as all the column.push() we had before, but in one place :)